osquery/tables/system/windows/bitlocker_info.cpp

```diff
@@ -21,19 +21,22 @@ namespace tables {
 QueryData genBitlockerInfo(QueryContext& context) {
   Row r;
   QueryData results;
-  WmiRequest wmiSystemReq(
-      "SELECT * FROM Win32_EncryptableVolume",
-      (BSTR)L"ROOT\\CIMV2\\Security\\MicrosoftVolumeEncryption");
```

Comment:
Ah actually, I spoke too soon. You'll want to leave this explicit cast here. I know it seems a bit redundant, but removing this will break VS conformance builds.

Reply:
Noted! Added that back in.